scripts/run_frontend_tests.py

```diff
@@ -115,8 +115,8 @@ def main(args: Optional[Sequence[str]] = None) -> None:
     if parsed_args.dtslint_only:
         return
 
-    if not parsed_args.skip_install:
-        install_third_party_libs.main()
+    # if not parsed_args.skip_install:
+    #     install_third_party_libs.main()
```

Comment:
Unexpected changes.

Reply:
There is an issue with `skip_install` flag i.e, even if you use it importing `install_third_party_libs` initializes installation somehow! I had done it to avoid installation steps, removed!